Jenkinsfile

```
@@ -282,6 +280,32 @@ pipeline {
         githubNotify(context: "${env.GITHUB_CHECK_ITS_NAME}", description: "${env.GITHUB_CHECK_ITS_NAME} ...", status: 'PENDING', targetUrl: "${env.JENKINS_URL}search/?q=${env.ITS_PIPELINE.replaceAll('/','+')}")
       }
     }
+    stage('Release') {
```

```suggestion
    stage('AfterRelease') {
```

this stage is not actually performing the release